### PR TITLE
Added announcement for Devi A S L.

### PR DESCRIPTION
**Pull request by bravegnu**

### CODE REVIEW

File: content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+She has been an active member of the PyCon India community since its
+inception, as a volunteer and as a speaker
+
+<br>
+
+![Devi A S L]({filename}/images/asldevi.jpg)
+
+<br>
+
+She can be reached at her twitter
```
Comment (astronomersiva) [inline content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md]:
```suggestion
She can be reached at her Twitter
```

File: content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Slug: announcing-keynote-speaker-for-pycon-india-2019-asldevi
+
+We are happy to announce that **Devi A S L** is a keynote speaker
+for PyCon India 2019!
+
+<!-- PELICAN_END_SUMMARY -->
+
+Devi is an independent software consultant, based in Bangalore.
+She is a long term Pythonista and her work spans diverse domains ranging
+from developing high-performance math libraries, cryptography to
```
Comment (astronomersiva) [inline content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md]:
```suggestion
from developing high-performance math libraries and cryptography to
```

File: content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+architecting complex web applications.
+
+Through her work that surpassed over a decade, she has experienced diverse
```
Comment (astronomersiva) [inline content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md]:
```suggestion
Through her work that has surpassed over a decade, she has experienced diverse
```

File: content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+work cultures at small startups, MNCs and remote teams where she has been
+through the lows and highs women in tech from different perspectives.
```
Comment (astronomersiva) [inline content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md]: > she has been through the lows and highs women in tech from different perspectives.

Doesn't this sound incomplete?

File: content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+She has been an active member of the PyCon India community since its
+inception, as a volunteer and as a speaker
```
Comment (astronomersiva) [inline content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md]:
```suggestion
inception, as a volunteer and a speaker.
```

File: content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md (hunk 4 above)
Comment (bravegnu) [reply content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md]: Yes. Will ask the speaker, about it. Can we merge with a known issue?

Comment (astronomersiva) [reply content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md]: Sure

**Comment by astronomersiva** [comment]
@bravegnu let's update the content once we hear back from the speaker. We should also replace this image with the optimised version in the site.

**Pull request merged**